app/pings.yaml

```diff
@@ -18,3 +18,14 @@ activation:
     - https://github.com/mozilla-mobile/fenix/pull/1707#issuecomment-486972209
   notification_emails:
     - fenix-core@mozilla.com
+
+installation:
+  description: >
+    Intended for counting user installs.
+  include_client_id: false
+  bugs:
+    - 7295
+  data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/1707#issuecomment-486972209
```

Comment:
This data review points to another ping type. Please do request a data review for this new collection. See [here](https://wiki.mozilla.org/Firefox/Data_Collection).

Reply:
Will add link to the proper data review answer when one will be available